packages/app-bridge-app/src/utilities/MessageBus.ts

```diff
@@ -53,7 +53,7 @@ export class MessageBus implements IMessageBus {
 
             setTimeout(() => {
                 reject(new TimeoutReachedError('operation'));
-            }, SUBSCRIBE_TIMEOUT * 10);
+            }, SUBSCRIBE_TIMEOUT * 300);
```

Comment:
@julianiff @imoutaharik Let's not increase it but allow per use-cases to increase.

Reply:
Sure, lets look into it when it fits our priorities. 👍